test/e2e/e2e_gitops_example_test.go

```diff
@@ -0,0 +1,103 @@
+
+		syncFileToRemoteServer(t, terraformOptions, keyPair, username, "../../build/zarf", fmt.Sprintf("/home/%s/build/zarf", username), "0700")
+		syncFileToRemoteServer(t, terraformOptions, keyPair, username, "../../build/zarf-init.tar.zst", fmt.Sprintf("/home/%s/build/zarf-init.tar.zst", username), "0600")
+		syncFileToRemoteServer(t, terraformOptions, keyPair, username, "../../build/zarf-package-gitops-service-data.tar.zst", fmt.Sprintf("/home/%s/build/zarf-package-gitops-service-data.tar.zst", username), "0600")
+	})
+
+	teststructure.RunTestStage(t, "TEST", func() {
+		terraformOptions := teststructure.LoadTerraformOptions(t, tmpFolder)
+		keyPair := teststructure.LoadEc2KeyPair(t, tmpFolder)
+
+		// Finally run the actual test
+		testGitopsExample(t, terraformOptions, keyPair, username)
+	})
+}
+
+func testGitopsExample(t *testing.T, terraformOptions *terraform.Options, keyPair *aws.Ec2Keypair, username string) {
+	// Run `terraform output` to get the value of an output variable
+	publicInstanceIP := terraform.Output(t, terraformOptions, "public_instance_ip")
+
+	// We're going to try to SSH to the instance IP, using the Key Pair we created earlier, and the user "ubuntu",
+	// as we know the Instance is running an Ubuntu AMI that has such a user
+	publicHost := ssh.Host{
+		Hostname:    publicInstanceIP,
+		SshKeyPair:  keyPair.KeyPair,
+		SshUserName: username,
+	}
+
+	// run `zarf init`
+	output, err := ssh.CheckSshCommandE(t, publicHost, fmt.Sprintf("sudo bash -c 'cd /home/%s/build && ./zarf init --confirm --components management,logging,gitops-service --host 127.0.0.1'", username))
+	require.NoError(t, err, output)
+
+	// Make sure Gitea comes up cleanly
+	output, err = ssh.CheckSshCommandE(t, publicHost, "timeout 300 bash -c 'while [[ \"$(curl -sfSL --retry 15 --retry-connrefused --retry-delay 5 -o /dev/null -w \"%{http_code}\" \"https://127.0.0.1/api/v1/user\")\" != \"401\" ]]; do sleep 1; done' || false")
+	require.NoError(t, err, output)
+
+	// Deploy the gitops example
+	output, err = ssh.CheckSshCommandE(t, publicHost, fmt.Sprintf("sudo bash -c 'cd /home/%s/build && ./zarf package deploy zarf-package-gitops-service-data.tar.zst --confirm'", username))
+	require.NoError(t, err, output)
+
+	// Check for full git repo mirror(foo.git) from https://github.com/stefanprodan/podinfo.git
+	output, err = ssh.CheckSshCommandE(t, publicHost, fmt.Sprintf("sudo bash -c 'cd /home/%s/build && git clone https://zarf-git-user:$(./zarf tools get-admin-password)@127.0.0.1/zarf-git-user/mirror__github.com__stefanprodan__podinfo.git'", username))
+	require.NoError(t, err, output)
+
+	// Check for tagged git repo mirror (foo.git@1.2.3) from https://github.com/defenseunicorns/zarf.git@v0.12.0
+	output, err = ssh.CheckSshCommandE(t, publicHost, fmt.Sprintf("sudo bash -c 'cd /home/%s/build && git clone https://zarf-git-user:$(./zarf tools get-admin-password)@127.0.0.1/zarf-git-user/mirror__github.com__defenseunicorns__zarf.git'", username))
```

Comment:
This is the same exact thing as line 77. Did you mean to include some tag or have something different?

Reply:
That's behaving correctly and just checking if that repo exists in gitea and got mirrored correctly after the gitops example install.

Reply:
Oops you're right. I must have been having a brain fart while looking at it before.